radix-engine/src/kernel/track.rs

```diff
@@ -616,6 +616,12 @@ impl<'s> FinalizingTrack<'s> {
                 to_persist.insert(id, (loaded.substate.to_persisted(), old_version));
             }
 
+            // TODO: Is there a better way to include this?
+            if matches!(next_epoch, Some((_, 1))) {
+                self.new_global_addresses
+                    .insert(0, Address::Package(PACKAGE_LOADER));
+            }
```

Comment:
We can create a package object for `Package` package.

Reply:
sweet!